votor/src/voting_utils.rs

```diff
@@ -63,11 +65,20 @@ pub enum BLSOp {
 
 #[derive(Debug, Error)]
 pub enum VoteError {
-    #[error("Unable to generate bls vote message")]
-    GenerationError(Box<GenerateVoteTxResult>),
+    #[error("Unable to generate bls vote message, transient error: {0:?}")]
+    GenerationInfo(Box<GenerateVoteTxResult>),
+
+    #[error("Unable to generate bls vote message, configuration error: {0:?}")]
+    GenerationWarning(Box<GenerateVoteTxResult>),
```

Comment:
I think coupling these errors to the log level isn't the best.

I would propose naming `TransientGeneration` and `InvalidConfig`

Reply:
I assume you mean TransientError, changed.

Reply:
yeah, `TransientError` works